components/admin_console/admin_definition.jsx

```diff
@@ -3468,6 +3468,10 @@ export default {
                 'admin.plugin.installedDesc',
                 'admin.plugin.uploadDesc',
                 'admin.plugin.uploadDisabledDesc',
+                'admin.plugin.enableMarketplace',
+                'admin.plugin.enableMarketplaceDesc',
```

Comment:
Is this being used somewhere? It seems that with the `helpText` this `*Desc` setting is not necessary?

Reply:
@marianunez 	are you referring to the string `admin.plugin.enableMarketplaceDesc` or why is `admin.plugin.enableMarketplaceDesc` in this list?

Reply:
Yes, why is admin.plugin.enableMarketplaceDesc in this list? I thought the main component covered it with the help text tag

Reply:
@marianunez  this is so that the the `*Desc` strings are searchable, however not sure how it is actually working, because these are the wrong ids 🤦‍♂ . But testing it, it does seem to work.. no idea how 🤔

Reply:
good catch btw, the ids were indeed wrong :P. Just updated them.